result/dhcpv6client.yaml

```diff
@@ -0,0 +1,147 @@
+openapi: 3.0.3
+info:
+  title: DHCPv6 results models
+  version: ^0.0.0
+components:
+  schemas:
+    Dhcpv6Client.Metrics.Request:
+      description: >-
+        The request to retrieve DHCPv6 per client metrics/statistics.
+      type: object
+      properties:
+        client_names:
+          description: >-
+            The names of DHCPv6 clients to return results for.
+            An empty list will return results for all DHCPv6 client.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Dhcpv6client/properties/name"
+          x-field-uid: 1
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned except for
+            any result_groups. The name of the DHCPv6 client cannot be excluded.
+          type: array
+          items:
+            type: string
+            x-enum:
+              solicits_tx:
+                x-field-uid: 1
+              advertisements_rx:
+                x-field-uid: 2
+              advertisements_ignored:
+                x-field-uid: 3
+              requests_tx:
+                x-field-uid: 4
+              requests_naked:
```

Comment:
nacks_received might be better similar to dhcpv4 , seems to be missing acks_received ?

Reply:
updated